hack/update-codegen.sh

```diff
@@ -28,21 +28,20 @@ fi
 CODEGEN_PKG="$GOPATH/pkg/mod/k8s.io/code-generator@${CODEGEN_VERSION}"
 
 # To avoid permission denied error
-chmod +x "${CODEGEN_PKG}/generate-groups.sh"
-chmod +x "${CODEGEN_PKG}/generate-internal-groups.sh"
+chmod +x "${CODEGEN_PKG}/kube_codegen.sh"
 
 # We can not generate client-go for v1alpha1 and v1beta1 and add them to the same directory.
 # So, we add each to a separate directory.
 # Generating files for v1alpha1
-"${CODEGEN_PKG}/generate-groups.sh" \
+"${CODEGEN_PKG}/kube_codegen.sh" \
     "deepcopy,client,informer,lister" \
     "github.com/kserve/kserve/pkg/clientv1alpha1" \
     "github.com/kserve/kserve/pkg/apis" \
     "serving:v1alpha1" \
     --go-header-file "${KUBE_ROOT}/hack/boilerplate.go.txt"
 
 # Generating files for v1beta1
-"${CODEGEN_PKG}/generate-groups.sh" \
+"${CODEGEN_PKG}/kube_codegen.sh" \
     "deepcopy,client,informer,lister" \
     "github.com/kserve/kserve/pkg/client" \
     "github.com/kserve/kserve/pkg/apis" \
```

Comment:
I think the usage of the script is also changed
see https://github.com/kubernetes/sample-controller/blob/master/hack/update-codegen.sh

Reply:
I haven't checked the details but this seems like a drop-in replacement since there's no diff generated after running this script.